open_dread_rando/files/schema.json

```diff
@@ -552,6 +560,57 @@
                     "type": "null"
                 }
             ]
+        },
+        "item_id": {
+            "type": "string",
+            "enum": [
+                "ITEM_NONE",
+                "ITEM_WEAPON_WIDE_BEAM",
+                "ITEM_WEAPON_PLASMA_BEAM",
+                "ITEM_WEAPON_WAVE_BEAM",
+                "ITEM_WEAPON_HYPER_BEAM",
+                "ITEM_WEAPON_CHARGE_BEAM",
+                "ITEM_WEAPON_DIFFUSION_BEAM",
+                "ITEM_WEAPON_GRAPPLE_BEAM",
+                "ITEM_WEAPON_SUPER_MISSILE",
+                "ITEM_WEAPON_ICE_MISSILE",
+                "ITEM_MULTILOCKON",
+                "ITEM_OPTIC_CAMOUFLAGE",
+                "ITEM_GHOST_AURA",
+                "ITEM_SONAR",
+                "ITEM_VARIA_SUIT",
+                "ITEM_GRAVITY_SUIT",
+                "ITEM_HYPER_SUIT",
+                "ITEM_MORPH_BALL",
+                "ITEM_WEAPON_BOMB",
+                "ITEM_WEAPON_LINE_BOMB",
+                "ITEM_WEAPON_POWER_BOMB",
+                "ITEM_MAGNET_GLOVE",
+                "ITEM_SPEED_BOOSTER",
+                "ITEM_DOUBLE_JUMP",
+                "ITEM_SPACE_JUMP",
+                "ITEM_SCREW_ATTACK",
+                "ITEM_ENERGY_TANKS",
+                "ITEM_LIFE_SHARDS",
+                "ITEM_WEAPON_MISSILE_CURRENT",
+                "ITEM_WEAPON_MISSILE_MAX",
+                "ITEM_WEAPON_POWER_BOMB_CURRENT",
+                "ITEM_WEAPON_POWER_BOMB_MAX",
+                "ITEM_FLOOR_SLIDE",
+                "ITEM_METROIDNIZATION",
+                "ITEM_RANDO_ARTIFACT_1",
+                "ITEM_RANDO_ARTIFACT_2",
+                "ITEM_RANDO_ARTIFACT_3",
+                "ITEM_RANDO_ARTIFACT_4",
+                "ITEM_RANDO_ARTIFACT_5",
+                "ITEM_RANDO_ARTIFACT_6",
+                "ITEM_RANDO_ARTIFACT_7",
+                "ITEM_RANDO_ARTIFACT_8",
+                "ITEM_RANDO_ARTIFACT_9",
+                "ITEM_RANDO_ARTIFACT_10",
+                "ITEM_RANDO_ARTIFACT_11",
+                "ITEM_RANDO_ARTIFACT_12"
+            ]
```

Comment:
`ITEM_MAX_LIFE` and `ITEM_CURRENT_LIFE` should be valid as well I think

Reply:
Does these two work properly if a pickup uses them?

Reply:
I'm not aware of any reason they shouldn't. The only pickups that grant them atm (etanks/eparts) have special handling, but I can't think of any reason the patcher shouldn't accept other pickups granting these resources without the special handling